static/app/components/replays/breadcrumbs/utils.tsx

```diff
@@ -3,12 +3,26 @@ import {BreadcrumbType, Crumb} from 'sentry/types/breadcrumbs';
 /**
  * Generate breadcrumb descriptions based on type
  */
-export function getDescription(crumb: Crumb) {
+export function getDescription(crumb: Crumb, startTimestampMs?: number) {
+  if (crumb.data && 'action' in crumb.data) {
+    switch (crumb.data.action) {
```

Comment:
Can we remove the outer if condition if we use opt chaining
```suggestion
    switch (crumb.data?.action) {
```

Reply:
Typescript isn't happy with it because action isn't available on all crumb types